windows/RNSVG/ImageView.cpp

```diff
@@ -260,10 +284,57 @@ IAsyncOperation<InMemoryRandomAccessStream> ImageView::GetImageInlineDataAsync(I
     memoryStream.Seek(0);
 
     co_return memoryStream;
-  } catch (winrt::hresult_error const &) {
+  } catch (hresult_error const &) {
     // Base64 decode failed
   }
 
   co_return nullptr;
 }
+
+com_ptr<IWICBitmapSource> ImageView::wicBitmapSourceFromStream(InMemoryRandomAccessStream const &results) {
+  if (!results) {
+    return nullptr;
+  }
+
+  com_ptr<IWICImagingFactory> imagingFactory{nullptr};
```

Comment:
Doesn't need nullptr init? (seeing this happen inconsistently, wondering if there's some reason why)

Reply:
no reason. I'm just inconsistent 🙃